modules/mbedtls/CMakeLists.txt

```diff
@@ -107,7 +107,7 @@ zephyr_interface_library_named(mbedTLS)
 
     zephyr_library_named(mbedTLSCrypto)
 
-    if (MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
@joerchan FYI. cmake file depends on the definition that is declared in the header file. Obviously, the header file is not available for cmake to choose files. That means mbedTLS PSA isn't possible to build with config files that are in Zephyr repo.